common/rpc/grpc.go

```diff
@@ -83,7 +83,7 @@ const (
 // The hostName syntax is defined in
 // https://github.com/grpc/grpc/blob/master/doc/naming.md.
 // dns resolver is used by default
-func Dial(hostName string, tlsConfig *tls.Config, logger log.Logger, interceptors ...grpc.UnaryClientInterceptor) (*grpc.ClientConn, error) {
```

Comment:
There's no use case for interceptors anymore; but there is for dial options now.

Reply:
Wondering if users who use the server as a library had a way to inject interceptors. I didn't read closely enough to tell.

Reply:
It appears this function is exported only to be used in a test. The only production code invocations are themselves unexported.